tests/common.py

```diff
@@ -51,64 +64,86 @@ def make_basic_auth(username, password):
 
 @pytest.fixture(scope="session")
 def mongo():
-    return MongoClient('mender-mongo:27017')
+    return MongoClient("mender-mongo:27017")
+
 
 def mongo_cleanup(mongo):
     dbs = mongo.database_names()
-    dbs = [d for d in dbs if d not in ['local', 'admin', 'config']]
+    dbs = [d for d in dbs if d not in ["local", "admin", "config"]]
```

Comment:
A lot of changes in this diff are due to the black substitution `" -> '`.
Not a big deal, indeed. Just one thing: maybe running black first, then changing the code in a separate commit makes the review easier? Don't know...

Reply:
ahh, sorry. I've set up my editor to run `black` formatter on save. If there are a lot of such changes I can remove them from this PR.

Reply:
It is okay, don't worry!